src/events/cleanup_emojis.ts

```diff
@@ -0,0 +1,32 @@
+import { DURATION_TMP_EMOJI } from '@constants';
+import type { BotEvent } from '@events';
+
+/*
+ * @event       - onReady - Cleanup temporary emojis
+ * @description - Gestion de la suppression des emojis temporaires
+ * @permission  - None
+ */
+export const CLEANUP_EMOJIS: BotEvent = {
+    name: 'Clean up temporary emojis',
+    listenTo: 'ready',
+    async execute(client) {
+        // Gestion de la suppression des emojis temporaires
+        const guilds = await client.guilds.fetch();
+        for await (const oauth2guild of guilds.values()) {
+            const guild = await oauth2guild.fetch();
+            const emojis = await guild.emojis.fetch();
+            for await (const emoji of emojis.values()) {
+                if (!emoji.name?.startsWith('tmp_')) {
+                    continue;
+                }
+                const timestamp = /tmp_(\d+)/.exec(emoji.name)?.[1];
+                if (!timestamp) {
+                    continue;
+                }
+                const diff = Math.max(0, DURATION_TMP_EMOJI - (Date.now() - Number(timestamp)));
```

Comment:
```suggestion
                const diff = Math.max(0, DURATION_TMP_EMOJI - Date.now() - timestamp);
```

Reply:
timestamp is a string, not a number. It doesn't work

Reply:
![Screenshot_2024-02-20-17-26-07-97_e4424258c8b8649f6e67d283a50a2cbc](https://github.com/la-zad/Zadier/assets/28602203/6efdc45f-502f-47a5-ac4b-24f22cb2aa0e)
Welcome to JS

Reply:
lol, you forget about eslint. he cries like a bitch

Reply:
You can try 
`Math.max(0, DURATION_TMP_EMOJI - Date.now() - +timestamp);`
Which should work